wallaby.js

```diff
@@ -1,10 +1,15 @@
-module.exports = function () {
+export default function wallabyConfig () {
```

Comment:
Currently Wallaby says this is supported, but it never loads when choosing this config